core/src/test/java/org/elasticsearch/search/builder/SearchSourceBuilderTests.java

```diff
@@ -216,16 +217,10 @@ protected final SearchSourceBuilder createSearchSourceBuilder() throws IOExcepti
             }
         }
         if (randomBoolean()) {
-            // NORELEASE make RandomQueryBuilder work outside of the
-            // AbstractQueryTestCase
-            // builder.query(RandomQueryBuilder.createQuery(getRandom()));
-            builder.query(QueryBuilders.termQuery(randomAsciiOfLengthBetween(5, 20), randomAsciiOfLengthBetween(5, 20)));
+            builder.query(RandomQueryBuilder.createQuery(getRandom()));
         }
         if (randomBoolean()) {
-            // NORELEASE make RandomQueryBuilder work outside of the
-            // AbstractQueryTestCase
-            // builder.postFilter(RandomQueryBuilder.createQuery(getRandom()));
-            builder.postFilter(QueryBuilders.termQuery(randomAsciiOfLengthBetween(5, 20), randomAsciiOfLengthBetween(5, 20)));
+            builder.postFilter(RandomQueryBuilder.createQuery(getRandom()));
```

Comment:
Agreed, this is not intended as full-blown solution to generate all possible random queries, also it does not address #14414 which will be a bigger effort. However, this change makes it possible to at least run the SearchSourceBuilderTests with a small set of randomized queries, which is a step forward and improves on the current situation. Also, I think we don't need to test e.g. SearchSourceBuilderTests with all possible query builders for the kinds of test we are doing here (roundtrip serialization etc...).